repos/system_upgrade/el7toel8/actors/selinux/selinuxapplycustom/tests/component_test_.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
Please rename the file back to component_test.py

import pytest and disable tests by using:

```python
@pytest.mark.skip(reason='Test disabled because it would modify the system')
def test_SELinuxApplyCustom(current_actor_context):
...
```

Reply:
Thank you. Updated.